swing-lib/src/main/java/org/triplea/swing/SwingComponents.java

```diff
@@ -406,11 +406,16 @@ public static void showDialogWithLinks(final DialogWithLinksParams params) {
 
     // parentComponent == null to avoid pop-up from appearing behind other windows
     final Component parentComponent = null;
```

Comment:
If you ask me we could inline this null value and keep the comment

Reply:
I somewhat wonder if the comment is actually correct and if previously the wrong parent component was simply being used. It could be that using a null parent component is always 'safer' even if the pop-up is not centered.

The assigning to a variable for a null value for the sake of adding a label is a bit old style compared to using a builder or something similar. In IDEA with the parameters being labelled it's far less useful, though still debatable as some would argue that the code should be understandable even if you are not using an IDE (which means you can infer what the variables are by looking at the usage and not have to look at the method definition).